test/integration/json.test.js

```diff
@@ -26,7 +26,8 @@ describe('model', () => {
       it('should tell me that a column is json', async function() {
         const table = await this.sequelize.queryInterface.describeTable('Users');
         // expected for mariadb 10.4 : https://jira.mariadb.org/browse/MDEV-15558
-        if (dialect !== 'mariadb') {
+        // oracledb 19c doesn't support JSON and the DB datatype is BLOB
+        if (dialect !== 'mariadb' && dialect !== 'oracle') {
```

Comment:
can you add an expect to check that the datatype is BLOB? (mariadb should have done the same, this will eventually be fixed)

Reply:
Done